Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -132,6 +149,11 @@ public static void DebugMessage(string message, bool IsError = false)
         #endregion
 
         #region Authentication
+        /// <summary>
+        /// Verify the player's steam identity with the server.
+        /// </summary>
+        /// <param name="steamSessionTicket">A steamSessionTicket in string-format</param>
+        /// <param name="onComplete">onComplete Action</param>
```

Comment:
```suggestion
        /// <param name="onComplete">onComplete Action for handling the response of type LootLockerVerifyResponse</param>
```

Reply:
I'll rewrite all of these "onComplete Action" to include the name of the response, and make it as a single commit, so won't comment more on these ones.